docs/EXTENDED-ARCHIVES.md

```diff
@@ -0,0 +1,344 @@
+Etcher Extended Archives
+========================
+
+Etcher Extended Archives are ZIP files including an image and a special `.meta`
+directory including information that describes the image, device type, and
+vendor in detail, enabling Etcher to make use of such information to deliver
+advanced features and a more pleasant experience to the user.
+
+**WARNING: At the moment of this writing, only a small subset of the features
+discussed in this document have been implemented in the product.**
+
+A typical layout for an extended archive might look something like this:
+
+```
+Raspbian-Jessie.zip
+├── .meta
+│   └── manifest.json
+│   └── instructions.markdown
+│   └── device-type.jpg
+│   └── vendor-logo.svg
+│   └── raspbian-jessie.img.bmap
+└── raspbian-jessie.img
+```
+
+The actual image file, `raspbian-jessie.img`, lives in the top level of the
+archive, and a special `.meta` directory contains many files, which will be
+explained in detail in the remaining of this document.
+
+The most important file inside `.meta` is `manifest.json`, which declares
+information about the image, the target device type, and the vendor, and
+references all the other files living inside the `.meta` directory. This file
+may contain the following top level properties:
+
+- `vendor`: Information about the image vendor
+- `device`: Information about the image device type
+- `image`: Information about the image itself
+
+For example:
+
+```json
+{
+  "vendor": { ... },
+  "device": { ... },
+  "image": { ... }
+}
+```
+
+Each of the top level properties are described in detail below.
+
+Vendor
+------
+
+> A vendor represents a manufacturer of a device type and potential publisher
+  of official images.
```

Comment:
it's the publisher. In fact, we may want to rename "vendor" to "publisher" to avoid confusion.